fe/fe-core/src/test/java/com/starrocks/planner/MaterializedViewTest.java

```diff
@@ -120,11 +120,11 @@ public void testFilter0() {
         String mv = "select empid + 1 as col1 from emps where deptno = 10";
         testRewriteOK(mv, "select empid + 1 from emps where deptno = 10");
         testRewriteOK(mv, "select max(empid + 1) from emps where deptno = 10");
-        testRewriteFail(mv, "select max(empid) from emps where deptno = 10");
+        testRewriteOK(mv, "select max(empid) from emps where deptno = 10");
```

Comment:
add more detail messages to display the content after rewrite? 

like this:
```
    // test union all
        testRewriteOK("select c1, c3, c2 from test_base_part where c3 < 3000")
                .contains("UNION")
                .contains("TABLE: partial_mv_6\n" +
                        "     PREAGGREGATION: ON\n" +
                        "     partitions=4/5\n" +
                        "     rollup: partial_mv_6\n" +
                        "     tabletRatio=8/8")
                .contains("TABLE: test_base_part\n" +
                        "     PREAGGREGATION: ON\n" +
                        "     PREDICATES: 10: c3 > 1999\n" +
                        "     partitions=2/5\n" +
                        "     rollup: test_base_part\n" +
                        "     tabletRatio=4/4");
```

Reply:
fixed